### PR TITLE
Use paratest for parallel test execution

### PR DESCRIPTION
**Pull request by manuelmeister**
To speed up our api tests, I've parallelized them with paratest. It works quite good. Has to be tested, if it actually tests, as the duration seems fishy

### CODE REVIEW

**Review by usu — APPROVED** [review]
Nice! 🚀 

Can I also use this to run tests in parallel on my local machine?

**Comment by manuelmeister** [comment]
@usu Yes, but you would need to setup the databases

**Comment by manuelmeister** [comment]
I realized that from https://github.com/ecamp/ecamp3/commit/39410e3045264ed1b731b149591c234f1cec6ac8 (update phpunit from v10.5.20 to v10.5.21, main feature: https://github.com/sebastianbergmann/phpunit/pull/5861) the test only takes 3m 19s, instead of 14m 34s.
With my PR it is again reduced to 1m 22s.

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -175,15 +175,15 @@ jobs:
...
+      DATABASE_URL: postgresql://ecamp3:ecamp3@localhost:5432/ecamp3?serverVersion=15&charset=utf8
 
     services:
       postgres:
         image: 'postgres:15-alpine'
         env:
-          POSTGRES_DB: 'ecamp3test'
           POSTGRES_PASSWORD: 'ecamp3'
           POSTGRES_USER: 'ecamp3'
+          POSTGRES_DB: 'ecamp3test'
```
Comment (BacLuc) [inline .github/workflows/continuous-integration.yml]: please revert this unnecessary change.
Else future devs might think that this change is needed.

File: api/config/packages/dev/doctrine.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+doctrine:
+  dbal:
+    # "TEST_TOKEN" is typically set by ParaTest
+    dbname_suffix: 'dev%env(default::TEST_TOKEN)%'
```
Comment (BacLuc) [inline api/config/packages/dev/doctrine.yaml]: i don't think you need the test token for dev

**Review by BacLuc — APPROVED** [review]
Please revert the unnecessary change and the change in dev/doctrine.yaml, and i am happy.

Tested it locally, and it ran out of memory when it tried to merge the coverage.
But most of us will anyway not run the tests locally anymore, it takes too much time.

But running the tests locally without paratest still works

**Comment by BacLuc** [comment]
It seems to be stable, tested it 20 times:

image

**Comment by BacLuc** [comment]
@manuelmeister 
will you finish this change or should i take over?

File: api/config/packages/dev/doctrine.yaml (hunk 2 above)
Comment (manuelmeister) [reply api/config/packages/dev/doctrine.yaml]: This is needed

Comment (BacLuc) [reply api/config/packages/dev/doctrine.yaml]: you need to add the suffix, but you don't need a test token.
ParaTest will never set the TEST_TOKEN for the dev environment, only in the test environment.

```suggestion
    dbname_suffix: 'dev'
```

**Comment by manuelmeister** [comment]
@BacLuc done

**Review by BacLuc — APPROVED** [review]
:heart:

**Pull request merged**